src/axom/primal/operators/detail/winding_number_impl.hpp

```diff
@@ -311,30 +311,70 @@ double stokes_winding_number(const Point<T, 3>& query,
 
     // Compute one of three vector field line integrals depending on
     //  the orientation of the original surface, indicated through ax.
-    if(ax == SingularityAxis::x)
+    switch(ax)
+    {
+    case(SingularityAxis::x):
     {
       quadrature += quad_rule.IntPoint(q).weight *
         (node[2] * node[0] * node_dt[1] - node[1] * node[0] * node_dt[2]) /
         (node[1] * node[1] + node[2] * node[2]) / node_norm;
+      break;
     }
```

Comment:
Minor: If there are no new variables defined, the curly brackets for this switch statement are not needed.

Reply:
Noted! Originally wasn't sure if this fell under the style guidelines for one-liners.